### PR TITLE
HCAP-1105 | MultiOrg | Display Hired Participant for same site users

### PR DESCRIPTION
**Pull request by fwpushan**

### CODE REVIEW

**Comment by fwpushan** [comment]
Please ignore Sonar scan. The alert is generated by test code.

File: server/services/participants-helper.js [hunk 1]
```diff
@@ -183,15 +183,28 @@ class FieldsFilteredParticipantsFinder {
...
+              },
+            }
+          : {
+              [employerSpecificJoin]: {
+                type: 'LEFT OUTER',
+                relation: collections.PARTICIPANTS_STATUS,
+                on: {
+                  participant_id: 'id',
+                  current: true,
+                  employer_id: user.id,
```
Comment (sunghwan-park-fw) [inline server/services/participants-helper.js:204]: Is this correct that employer_id on-clause is for the status filter without 'hired' condition?

File: server/services/participants-helper.js [hunk 2]
```diff
@@ -237,6 +250,13 @@ class FieldsFilteredParticipantsFinder {
         }),
       });
 
+      if (statusFilters && statusFilters.includes('hired')) {
```
Comment (sunghwan-park-fw) [inline server/services/participants-helper.js:253]: Line 186 doesn't check if statusFilter is defined.

**Review by sunghwan-park-fw — COMMENTED** [review]
@fwpushan,
I left two comments.

**Review by sunghwan-park-fw — APPROVED** [review]

**Pull request merged**